index.rst

```diff
@@ -490,6 +490,13 @@ What's new in PyTorch tutorials?
    :link: intermediate/torchserve_with_ipex
    :tags: Model-Optimization,Production
 
+.. customcarditem::
+   :header: Introduction to nvFuser
+   :card_description: An introduction to nvFuser TODO
+   :image: _static/img/thumbnails/cropped/60-min-blitz.png
+   :link: intermediate/nvfuser_intro_tutorial
+   :tags: Model-Optimization
```

Comment:
Added this intro tutorial to `Model-Optimization` as it seems to be the best fit